homeassistant/components/plaato/strings.json

```
@@ -3,12 +3,12 @@
     "step": {
       "user": {
         "title": "Set up the Plaato Webhook",
-        "description": "Are you sure you want to set up the Plaato Airlock?"
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
       }
     },
     "abort": {
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
-      "not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive messages from Plaato Airlock."
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
```

Please revert this change as the reference string does not reflect the original error message.
```suggestion
      "not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive messages from Plaato Airlock."
```